tests/IntegrationTest.php

```diff
@@ -45,4 +46,32 @@ public function testEvents()
             $statistics[0]
         );
     }
+
+    public function testSince()
+    {
+        $now = CarbonImmutable::now();
+        $yesterday = $now->subDay();
+        Carbon::setTestNow($yesterday);
+
+        $collector = $this->app->get(StatisticsCollector::class);
+        $collector->event('guid', 'event', 'object', 'item', ['extra1', 'extra2']);
+
+        // Ensure that an event is available.
+        $response = $this->json('PATCH', 'statistics');
+        $this->assertEquals(1, count($response->json()));
+
+        // Event should still be available as it is created at the same time as
+        // the since argument.
+        $response = $this->json('PATCH', 'statistics?since=' . urlencode($yesterday->toAtomString()));
+        $this->assertEquals(1, count($response->json()));
+
+        // No event should be returned as it is before the since argument.
+        // Also this should trigger a cleanup causing the event to be deleted.
```

Comment:
The comment suggest that we're checking that an event is fired, but we're not really. There's `fake` methods on the Event/Job facades that activates assertion methods for checking if event/job was fired, but I don't know if they're useable here.

But as this is an integration test, it might not make sense to test here (it's really not about integration), but let it be tested in the controller test.

Reply:
:+1: That is exactly what we are doing in the controller test now.